ci/Jenkinsfiles/build.groovy

```diff
@@ -183,7 +183,7 @@ pipeline {
                   // customEnvironment profile
                   sh 'touch /root/nuxeo-test-dev.properties'
                   retry(3) {
-                    sh "mvn ${MAVEN_CLI_ARGS} -pl :nuxeo-coldstorage -Dcustom.environment=dev test"
+                    sh 'mvn -B -nsu -pl :nuxeo-coldstorage -Dcustom.environment=dev -Dcustom.environment.log.dir=target-dev test'
```

Comment:
Why removing `${MAVEN_CLI_ARGS}` here?
Also, do we need to add `-Dcustom.environment.log.dir=target-dev ` in a non temporary commit?
If so, do we also need a custom log dir for the 'Backend - MongoDB' and 'Backend - PostgreSQL' stages?

Reply:
This change is an error which is inside the tmp commit, the final change will be:
```java
sh "mvn ${MAVEN_CLI_ARGS} -pl :nuxeo-coldstorage -Dcustom.environment=dev test"
```

Reply:
> Also, do we need to add `-Dcustom.environment.log.dir=target-dev ` in a non temporary commit? If so, do we also need a custom log dir for the 'Backend - MongoDB' and 'Backend - PostgreSQL' stages?

No, the `custom.environment.log.dir` is defined by nuxeo-lts repository based on the `custom.environment` property.